### DISCUSSION TITLE
How to set up git backup ?

### DISCUSSION THREAD

**Discussion by M1st3r3** [requarks/wiki#6324] [Help / Questions] [answered]
Ive tried everything but it dont seems to works i just wanna push the wiki to git not even pull or bi directionel only push. if someone could help me :)

**Comment by M1st3r3** [comment]
Update its stuck on the config.yml file in my cmd

**Comment by NGPixel** [comment, marked as answer]
Have you [read the docs](https://docs.requarks.io/storage/git)?

**Reply by M1st3r3** [reply]
it ok i made it work with the docs thanks
